### ISSUE TITLE
Library dont seem to dowload with gradle dependency

### ISSUE THREAD

**Issue by iambuntoo** [ezralazuardy/orb#6]
**Describe the bug**
I added jitpack url in my project level gradle and your library dependency in my app level gradle file but when i try to use 
```
Orb.with(this).observe { response ->
    Log.d(this::javaClass.simpleName, response.toString())
    // do something awesome..
}
```
then when i try to auto-import it doesn't import any class then i dig in project view external libraries and seems like no classes downloaded
![Capture](https://user-images.githubusercontent.com/18241005/92316830-d3b2a800-f012-11ea-9f92-8556f60b3514.png)

**To Reproduce**
1. Create a new project in Android Studio
2. Add dependency
3. Try to use

**Comment by ezralazuardy** [comment]
do you have add this in your project level gradle:

```
allprojects {
    repositories {
        maven { url 'https://jitpack.io' }
    }
}
```

and this in your module level gradle:

```
dependencies {
    implementation 'com.github.ezralazuardy:orb:$version' //replace '$version' with the latest version of orb
}
```

please make sure that you should replace the ```$version``` with the current latest version of orb. check it [here](https://github.com/ezralazuardy/orb/releases).

**Comment by ezralazuardy** [comment]
ok, I've try installing in new project and it's not installed properly. looks like there is something wrong in the jitpack repo. I'm confirming this issue now. thanks for reporting.

**Comment by ezralazuardy** [comment]
this bug have been fixed in [v0.0.3](https://github.com/ezralazuardy/orb/releases). please let me know if it's working in your project too.

**Comment by iambuntoo** [comment]
yeah its fix now thanks for the quick fix
